docs/package.json

```diff
@@ -51,5 +55,9 @@
     },
     "engines": {
         "node": ">=14.0.0"
-    }
+    },
+    "browserslist": [
+        "defaults",
+        "IE 11"
```

Comment:
Do we still need to support IE 11 at this point? I would strongly advocate against explicitly supporting it because:

- It is [end-of-life](https://blogs.windows.com/windowsexperience/2022/06/15/internet-explorer-11-has-retired-and-is-officially-out-of-support-what-you-need-to-know) as of earlier this year.
- Actively supporting it increases maintenance/support burden and potentially imposes development limitations.
- Typically, IE support would be more important for a consumer-facing site like Amazon where some fraction of potential customers could still be using very outdated software and the site has no control over their software. ESLint's site is largely developer-facing, and it's hard to imagine any ESLint users still using IE 11, especially considering ESLint has explicit requirements that users must be using a very modern version of Node. I don't see how anyone with a modern Node environment could still be depending on IE.

Reply:
I agree, but I added it based on the discussion in an earlier PR around removing fallbacks https://github.com/eslint/eslint/pull/16466#discussion_r1008641497

Reply:
It looks like previous discussion just assumed we wanted to maintain the status quo. I'm fine with maintaining the status quo in this PR, but my recommendation to stop explicitly supporting IE still stands (could be a good follow-up PR to consider) (CC: @nzakas).

Reply:
I generally feel like if we can get it for free, then let’s do it. In this case it seems like we can get IE11 support for free, so I don’t see any harm in keeping it.